Formula/distcc.rb

```diff
@@ -21,12 +25,24 @@ class Distcc < Formula
 
   depends_on PythonWithoutPPCRequirement
 
+  depends_on "autoconf" if build.head?
+  depends_on "automake" if build.head?
+  depends_on "gcc" if build.head?
+
   def install
     # Make sure python stuff is put into the Cellar.
     # --root triggers a bug and installs into HOMEBREW_PREFIX/lib/python2.7/site-packages instead of the Cellar.
-    inreplace "Makefile.in", '--root="$$DESTDIR"', ""
 
-    system "./configure", "--prefix=#{prefix}"
+    if build.head?
+      system "./autogen.sh"
+    end
+
+    inreplace "Makefile.in", '--root="$$DESTDIR"', ""
+    args = %W[
+      --prefix=#{prefix}
+    ]
+    args << "--without-libiberty" if build.head?
```

Comment:
This isn't working in HEAD, this library is not shipped by us in gcc and binutils, there is a number of bugs regarding this library. [Upstream recommends](https://gcc.gnu.org/onlinedocs/libiberty/Using.html) to ship it with project source code.